src/components/Button/ButtonLink/styled.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+export const ButtonCover = styled(Link)`
```

Comment:
This is much better, I naming it `StyledLink` instead of `ButtonCover` would be more suitable. Don't you think? :slightly_smiling_face:
```suggestion
export const StyledLink = styled(Link)`
```